app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,46 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all.order(:completed, :created_at)
+  end
+
+  def show
+    @task = Task.find_by(id: params[:id])
+  end
```

Comment:
What happens if the task isn't found? You should include error-handling code in all of the actions that interact with a single task.